back/boxtribute_server/business_logic/box_transfer/agreement/mutations.py

```diff
@@ -43,7 +44,6 @@ def resolve_accept_transfer_agreement(*_, id):
     )
     bases = retrieve_transfer_agreement_bases(transfer_agreement=agreement, kind=kind)
     authorize(permission="transfer_agreement:edit", base_ids=[b.id for b in bases])
```

Comment:
If I understand the `authorize` function correctly, then the user only needs to have access to one of the bases to accept a transfer agreement.

Meaning if there is a transferAgreement from org 1 / base 1 to org 2 / all bases(2,3,4) then a coordinator of base 2 can accept the agreement for all three bases.

I tested this in the playground and it works. I'm not sure if this is the wanted behavior.

Reply:
I'd consider it an inconsistency in authorization. I changed this such that the user must be authorized for **all** relevant bases to accept the agreement.